Nikita Belyaev/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>New Jersey</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+</head>
+<body>
+
+    <article class="widget">
```

Comment:
Здесь я исходил из [спецификации](https://www.w3.org/TR/html5/sections.html#the-article-element).

> The article element represents a complete, or self-contained, composition in a document, page, application, or site.

Тег `article` вполне уместен, поскольку виджет является завершенным и самостоятельным элементом, содержание которого имеет смысл вне контекста страницы.

[HTML Living Standard](https://html.spec.whatwg.org/multipage/sections.html#the-article-element) поясняет

>This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an **interactive widget** or gadget, or any other independent item of content.

А [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) даже иллюстрирует применение этого тега погодным виждетом.

Reply:
Отличный аргумент спецификацией 👍